v2.0/create-table.md

```diff
@@ -152,10 +152,15 @@ In this example, we create a table with three columns. One column is the [primar
 (3 rows)
 ~~~
 
-### Create a Table with Secondary Indexes
+### Create a Table with Secondary and Inverted Indexes <span class="version-tag">New in v2.0</span>
 
-In this example, we create two secondary indexes during table creation. Secondary indexes allow efficient access to data with keys other than the primary key. This example also demonstrates a number of column-level and table-level [constraints](constraints.html).
+In this example, we create two secondary indexes and an inverted index during table creation. [Secondary indexes](indexes.html) allow efficient access to data with keys other than the primary key.
 
+<span class="version-tag">New in v2.0:</span> [Inverted indexes](inverted-indexes.html) allow efficient access to schemaless data.
```

Comment:
Since we have the callout in the header, I'd say this can just be a continuing of the paragraph above, perhaps like:

> Inverted indexes, which are new in v2.0, allow efficient access to the schemaless data in a `JSONB` column.

Reply:
done